examples/resources/grafana_oncall_route/resource.tf

```diff
@@ -2,13 +2,19 @@ data "grafana_oncall_slack_channel" "example_slack_channel" {
   name = "example_slack_channel"
 }
 
+data "grafana_oncall_integration" "example_integration_data_source" {
+  id = "CABCDEFGHI321"
+}
+
```

Comment:
hmm, I'd remove this one. The link between integration and route is already in this example file